File: app/src/main/java/com/tungnui/abccomputer/model/Category.java
package com.tungnui.abccomputer.model;

/**
 * Created by Nasir on 3/29/17.
 */

//TODO Need to add more category attributes for parsing more data

public class Category {
    public int id;
    public String name;
    public String image;
    public String description;
    public boolean isSelected;

    public Category(int id, String name, String image, String description, boolean isSelected) {
        this.id = id;
        this.name = name;
        this.image = image;
        this.description = description;
        this.isSelected = isSelected;
    }
}

File: app/src/main/java/com/tungnui/dalatlaptop/models/Category.kt
package com.tungnui.dalatlaptop.models

/**
 * Created by thanh on 28/09/2017.
 */

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName



data class Category(
        @SerializedName("id")
        @Expose
        var id: Int? = null,
        @SerializedName("name")
        @Expose
        var name: String? = null,
        @SerializedName("slug")
        @Expose
        var slug: String? = null,
        @SerializedName("parent")
        @Expose
        var parent: Int? = null,
        @SerializedName("description")
        @Expose
        var description: String? = null,
        @SerializedName("display")
        @Expose
        var display: String? = null,
        @SerializedName("image")
        @Expose
        var image: Image? = null,
        @SerializedName("menu_order")
        @Expose
        var menuOrder: Int? = null,
        @SerializedName("count")
        @Expose
        var count: Int? = null
)  {

    fun hasChildren():Boolean{
        return true;
    }
}
